api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/SqlCohortQueries.java

```diff
@@ -267,14 +280,69 @@ public CohortDefinition getDeadPersons() {
 		return commonCohortQueries.general("Dead persons", "SELECT person_id FROM person WHERE dead=1");
 	}
 	
+	// Start of TX_CURR
+	
+	// Looks for patients registered as START DRUGS (answer to question 1255 = ARV
+	// PLAN is 1256 = START DRUGS) in the first drug pickup (encounter type
+	// 18=S.TARV: FARMACIA) or follow up consultation for adults and children
+	// (encounter types 6=S.TARV: ADULTO SEGUIMENTO and 9=S.TARV: PEDIATRIA
+	// SEGUIMENTO) before or on end date
+	@DocumentedDefinition(value = "patientWithSTARTDRUGSObs")
+	public CohortDefinition getPatientWithSTARTDRUGSObsBeforeOrOnEndDate() {
```

Comment:
@rubailly why have you created these differently for TxCurr? Why only measuring `OnOrBefore` instead of before and after?

Reply:
Same as the first comment, it is because we want patients who are **currently** documented as "started on ART". they may have been started on ART a long time ago, we just want to know the current status, which is why we only use the endDate parameter.